src/app/public/src/modules/table-of-contents/table-of-contents-wrapper.component.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../styles/_variables';
+
+.stache-table-of-contents-wrapper {
+  background-color: white;
+  position: absolute !important;
+  right: 0;
+  top: 0;
+  bottom: 0;
+
+  .stache-table-of-contents-content {
+    width: $stache-sidebar-column-width;
+    padding-top: $stache-padding-half;
+    padding-bottom: $stache-padding-plus-half;
+    padding-right: $stache-padding-half;
+    max-height: 100%;
+
+    .stache-affix-top {
+      overflow-y: scroll;
```

Comment:
This would need to be `overflow-y: auto`

Reply:
Also should this be on the affix top? or on the `table-of-contents-content`

Reply:
I went ahead and tested it and it looks like it must be in the `stache-affix-top` element for the scrolling to trigger.